var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -119,19 +119,22 @@ def cmake_args(self):
 
     def setup_run_environment(self, env):
         env.prepend_path("PYTHONPATH", self.prefix.python)
-        env.prepend_path("LD_LIBRARY_PATH", self.spec["podio"].libs.directories[0])
+        env.prepend_path("LD_LIBRARY_PATH", self.spec["podio"].prefix.lib)
+        env.prepend_path("LD_LIBRARY_PATH", self.spec["podio"].prefix.lib64)
```

Comment:
I woul have suggested to do exactly the opposite for `edm4hep` above. Using `libs.directories[0]` is the recommended way to do this AFAIK, why switch back to the duplication here?

Reply:
Fixed